MSAL/src/requests/MSALRequestParameters.m

```diff
@@ -61,4 +68,14 @@ - (BOOL)setAuthorityFromString:(NSString *)authority
     return YES;
 }
 
+- (MSIDRequestParameters *)msidParameters
+{
+    MSIDRequestParameters *requestParameters = [[MSIDRequestParameters alloc] initWithAuthority:self.unvalidatedAuthority
+                                                                                    redirectUri:self.redirectUri.absoluteString
```

Comment:
interesting that redirectUri is URL, didn't know that

Reply:
According to [OAuth2](https://tools.ietf.org/html/rfc6749#page-72) it could be URN. I think it is better to use sting here, what do you think?

Reply:
I agree